docs/specification.md

```diff
@@ -227,7 +233,10 @@ Parameter (in Michelson)
 
 - Returns total supply for multiple tokens. It accepts a list of
   token ids and a callback which accepts a list of
-  `getTotalSupplyResponse` which is conequently passed to it.
+  `total_supply_response` which is conequently passed to it.
+
+- Since the contract supports only a single token type, `tokenId` must be 0.
+  It is passed because FA2 requires that.
 
 **getTokenMetadata**
```

Comment:
I suppose this will be unused due to single-token nature of our contract?

Reply:
It can be used to get metadata about this single token. And we must have an entrypoint with exactly this type because it's required by FA2.